.github/dependabot.yml

```diff
@@ -10,3 +10,38 @@ updates:
     directory: "/model-server"
     schedule:
       interval: "daily"
+  - package-ecosystem: "github-actions"
+    open-pull-requests-limit: 20
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "npm"
+    open-pull-requests-limit: 20
+    directory: "/"
+    schedule:
+      interval: "daily"
+    versioning-strategy: "increase"
```

Comment:
Some explanation on why this strategy was chosen would be good. The default `auto` would actually use `widen` for libraries, which we provide, so that clients are free to select an appropriate version from the allowed range when they use one of the libraries.

Reply:
I don't have a strong opinion on the versioning-strategy. If you think another one is more appropriate, I will change it. @odzhychko what do you think, since this affects components you developed?

Reply:
> `auto`	Try to differentiate between apps and libraries. Use increase for apps and widen for libraries.

> so that clients are free to select an appropriate version from the allowed range when they use one of the libraries.

I would use `auto`, because of what languitar said about libraries.

In the upcoming automatically created PRs we should check, if Dependabot appropriately differentiated between libraries and applications. If not, we would need to upgrade the version appropriately by ourselves. I did not see that we could override the `versioning-strategy` for specific libraries.